utils/biospecimen.js

```diff
@@ -673,6 +757,28 @@ const biospecimenAPIs = async (req, res) => {
         return res.status(200).json({data: response, code:200})
     }
 
+    else if(api === 'kitStatusToParticipant') {
+        if(req.method !== 'POST') {
+            return res.status(405).json(getResponseJSON('Only POST requests are accepted!', 405));
+        }
+        const requestData = req.body;
+        if(Object.keys(requestData).length === 0 ) return res.status(400).json(getResponseJSON('Request body is empty!', 400));
+        const { addKitStatusToParticipant } = require('./firestore');
+        const response = await addKitStatusToParticipant(requestData);
+        if(!response) return res.status(404).json(getResponseJSON('ERROR!', 404));
+        return res.status(200).json({message: `Success!`, code:200})
+    }
+
+    else if(api === 'getElgiblePtsForAssignment') {
+        if(req.method !== 'GET') {
+            return res.status(405).json(getResponseJSON('Only GET requests are accepted!', 405));
+        }
+        const { eligibleParticipantsForKitAssignment } = require('./firestore');
```

Comment:
Please consider consistent naming conventions between the endpoint and the function.